packages/meditrak-app/app/assessment/actions/submitSurvey.js

```diff
@@ -242,14 +267,27 @@ export const submitSurvey = (surveyId, userId, startTime, questions, shouldRepea
     analytics.trackEvent('Submit Survey', response);
   }
 
+  const isGeneratingQrCodes = qrCodes.length > 0;
```

Comment:
submitSurvey seems to be doing a lot now. I suggest it just dispatches a onAfterSurveySubmitted action and we pull all this logic out into there, what do you think?

Reply:
Or maybe just a function. Either way, minor.

Reply:
Yeah good point. I might break it into a function. The redux stuff is kinda confusing